docs/buildkitd.toml.md

```diff
@@ -106,6 +106,11 @@ insecure-entitlements = [ "network.host", "security.insecure" ]
   [worker.containerd.labels]
     "foo" = "bar"
 
+  # configure the containerd runtime
+  [worker.containerd.runtime]
+    runtime = "io.containerd.runc.v2"
+    options = { binary_name = "runc" }
```

Comment:
Why isn't this `BinaryName` ?

Reply:
Ah I was converting to JSON as an intermediate, but best to stay consistent with CRI here, IMO.